### PR TITLE
fix,chore,refactor(egen): minor changes to the automl training notebook.

### PR DESCRIPTION
**Pull request by sriramya2610**
This PR includes the following updates:

1. Adds Colab Enterprise link and other structural changes from the new template.
2. Changed REGION variable name to LOCATION.
3. Modifies the import file of flowers dataset.
4. Adds comments in cleanup section.
5. Removes os.getenv("IS_TESTING") from the cleanup section.
6. Contracts words like is not, do not, you are, does not, will not  etc.
7. Changes to second person sentences.
8. Changes future tense sentences to present tense.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/automl/get_started_automl_training.ipynb [hunk 1]
```diff
@@ -32,25 +32,27 @@
         "# Get started with AutoML Training\n",
```
Comment (kittyabs) [inline notebooks/official/automl/get_started_automl_training.ipynb:32]: lower case: "AutoML training"

File: notebooks/official/automl/get_started_automl_training.ipynb [hunk 2]
```diff
@@ -949,7 +915,7 @@
       "source": [
         "#### Undeploy the model\n",
         "\n",
-        "When you are done doing predictions, you undeploy the model from the `Endpoint` resouce. This deprovisions all compute resources and ends billing for the deployed model."
+        "When you're done doing predictions, you undeploy the model from the `Endpoint` resouce. This deprovisions all compute resources and ends billing for the deployed model."
```
Comment (kittyabs) [inline notebooks/official/automl/get_started_automl_training.ipynb:918]: delete "you" and fix typo ("resouce"):
- change to: "When you're done doing predictions, undeploy the model from the `Endpoint` resource."

File: notebooks/official/automl/get_started_automl_training.ipynb [hunk 3]
```diff
@@ -1351,7 +1317,7 @@
...
```
Comment (kittyabs) [inline notebooks/official/automl/get_started_automl_training.ipynb:1320]: delete "you" and fix typo ("resouce"):
- change to: "When you're done doing predictions, undeploy the model from the `Endpoint` resource."

File: notebooks/official/automl/get_started_automl_training.ipynb [hunk 4]
```diff
@@ -1851,7 +1817,7 @@
...
```
Comment (kittyabs) [inline notebooks/official/automl/get_started_automl_training.ipynb:1820]: delete "you" and fix typo ("resouce"):
- change to:  "When you're done doing predictions, undeploy the model from the `Endpoint` resource."

**Review by kittyabs — COMMENTED** [review]
numerous typos and other edits

**Review by katiemn — APPROVED** [review]

**Pull request merged**